src/levels.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash';
+
+export let levels = {
+  // https://tools.ietf.org/html/rfc3164 (multiplier 10)
+  emergency: 0,
+  alert: 10,
+  critical: 20,
+  error: 30,
+  warning: 40,
+  notice: 50,
+  informational: 60,
+  debug: 70,
+
+  // console
+  warn: 40, // warning
+  info: 60, // informational
+  trace: 90,
+
+  // alias
+  fatal: 0, // emergency
+  verbose: 70, // debug
+  silly: 80,
+
+
+  levelToLevelName: function(level) {
```

Comment:
is it a level? nope, so export as a separate function please and you can pass the levels as an argument